src/types/profile-derived.js

```diff
@@ -270,6 +277,16 @@ export type ActiveTabResourceTrack =
       +name: string,
     |};
 
+export type ActiveTabTimeline = {
+  mainTrack: ActiveTabMainTrack,
```

Comment:
Yeah, this is much more specific, I think this will make things easier. It would be good to add a small bit of documentation above the type explaining it.

Reply:
Yes, that's a good idea. Adding it.